docs/tutorials/ruby-driver-crud-operations.txt

```diff
@@ -130,6 +130,43 @@ object, or with ``Decimal128.from_string()``.
    price = BSON::Decimal128.from_string("428.79")
    # => BSON::Decimal128('428.79')
 
+Query Cache
+===========
+
+If the Ruby driver's QueryCache is enabled, it will cache find queries on
```

Comment:
And whenever "query cache" is used to refer to the concept and not specifically to the class in code, can you spell it as two words without special capitalization please.